docs: Edits to remove jargon (#42965)
angular/angular: aio/content/guide/testing-utility-apis.md

      The testing shims (`karma-test-shim`, `browser-test-shim`) call it for you
      so there is rarely a reason for you to call it yourself.

>       You can call this method _exactly once_. If you must change
>       this default in the middle of your test run, call `resetTestEnvironment` first.

Reviewer: ```suggestion
      Call this method _exactly once_. To change
      this default in the middle of a test run, call `resetTestEnvironment` first.
```
